### ISSUE TITLE
Even copy persistent leaves, pasted leaves are persistent=false

### ISSUE THREAD

**Issue by nishio** [EngineHub/WorldEdit#1992]
### WorldEdit Version

WorldEdit 7.2.8

### Platform Version

paper-1.18.1-68.jar

### Confirmations

- [X] I am using the most recent Minecraft release.
- [X] I am using a version of WorldEdit compatible with my Minecraft version.
- [X] I am using the latest or recommended version of my platform software.
- [X] I am NOT using a hybrid server, e.g. a server that combines Bukkit and Forge. Examples include Arclight, Mohist, and Cardboard.
- [X] I am NOT using a fork of WorldEdit, such as FastAsyncWorldEdit (FAWE) or AsyncWorldEdit (AWE)

### Bug Description

I put leaves. Those have `persistent=true` and do not decay as expected vanilla behavior.
Copy them with worldedit and paste. The pasted leaves have `persistent=false` and decay.

### Expected Behavior

Clipboard keeps `persistent` values.

### Reproduction Steps

1. put leaves
2. copy them with worldedit
3. paste them
4. press F3 to see its `persistent` value

**Comment by octylFractal** [comment]
Please provide your server logs.

**Comment by nishio** [comment]
There were no error logs.
And I'm sorry but I cannot reproduce the problem. I agree to close this issue.

Here are some more information
case A(described above): Leaves are persistent
case B: copy, schem save, schem load, paste:  Leaves are persistent
case C: create leaves with command (\hcyl): Leaves are not persistent. It can avoid by using `leaves[persistent=true]`
case D: what I actually did: copy, schem save(on Mac), schem load in another server(on Linux), paste
I feel it was complicated situation than I reported before.
